### PR TITLE
figma variables sync 

### PR DESCRIPTION
**Pull request by AnnaAres**
- updated grouping of the themes to allow syncing with figma variables
- added brown colour and accent colours
- fixed 1 wrong purple token
- updated emerson text tokens to match fs text tokens

### CODE REVIEW

**Comment by iryanclarke** [comment]
@AnnaAres We're going to have to do some dev tweaks in this MR, the filepath update that Token Studio did has borked our build pipelines 😄

File: dist/documentation/themes/dark.d.ts [hunk 1]
```diff
@@ -564,4 +564,37 @@ export const shadow : {base:{
...
+            
+          },sandBackgroundMuted:{
+            name: string,
+            hex: string,
+            
+          },sandBackgroundMutedHover:{
+            name: string,
+            hex: string,
+            
+          },sandBackgroundMutedActive:{
```
Comment (CCAyl) [inline dist/documentation/themes/dark.d.ts:596]: i wonder if we need an accent transform 🫠, so this is nested more. `accent.sand.backgroundMutedActive`


or alternatively, remove the `accent` level, but we prob wanna keep that

File: src/data/themes/dark.json [hunk 2]
```diff
@@ -832,10 +878,10 @@
   "opacity": {
     "overlay": {
       "backdrop": {
-        "value": 0.8,
+        "value": "0.8",
```
Comment (CCAyl) [inline src/data/themes/dark.json:881]: is this being a string gunna be an issue? 🤔

File: dist/documentation/themes/dark.d.ts (hunk 1 above)
Comment (iryanclarke) [reply dist/documentation/themes/dark.d.ts:596]: Yeah, I think we should probably nest one more, will likely be too many options, especially when we are wanting to possibly add more for things like `ApplePay` and such that have specific brand colors

File: package.json [hunk 3]
```diff
@@ -15,10 +15,10 @@
     "transform-core-light": "yarn token-transformer src/data src/transformed/transformed-core-light.json core-light",
```
Comment (iryanclarke) [inline package.json:15]: You should also do the version bump in here

File: src/data/themes/dark.json (hunk 2 above)
Comment (alexandra-lim) [reply src/data/themes/dark.json:881]: Oof this was a mistake I made in the custom.js file, will be fixed to be a number when built

File: dist/documentation/themes/dark.d.ts (hunk 1 above)
Comment (alexandra-lim) [reply dist/documentation/themes/dark.d.ts:596]: Unless we foresee these accent colours being used immediately, let's hold off on transforming them for now. It's a bit more challenging than expected, and isn't priority compared to getting DS4 out. I'll make a ticket for the backlog to tackle this

Comment (iryanclarke) [reply dist/documentation/themes/dark.d.ts:596]: Sounds good @alexandra-lim, agree on prioritizing getting something initial out for DS4 release

**Review by iryanclarke — APPROVED** [review]
This is looking good to go!

**Pull request merged**